implementation.md

```diff
@@ -1554,3 +1554,268 @@ In the `Video` and `Audio` objects, sellers must only include one form of floor
+```
+{
+    "user": {
+        "buyeruid": "fac13741-0648-436a-88cf-aceafdf45c9a",
+        "eids": [
+            {
+                "uids": [
+                    {
+                        "atype": "1"
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+
+##### Example 2. Resettable advertising ID generated by operating system
+```
+{
+    "device": {
+        "ifa": "df3a42f4-a388-43e9-a691-b0d308c963b4",
+        "eids": [
+            {
+                "uids": [
+                    {
+                        "atype": "2"
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+
+#### 7.12.4.2 Derrived IDs for a Single Property
+There exist scenarios in which a party may use an alternate approach to infer a DSP cookie user ID or IFA that is believed to apply or be related to the user, but cannot be directly observed. These scenarios should be conveyed using the below mechanism.
+
+##### Example 3: Request from Pub → SSP, user ID matched on hashed email, cookie not observable by SSP in the customary way
+
+```
+{
+    "user": {
+        "eids": [
+            {
+            "inserter": "pub.com",
+            "matcher": "graphvendor.com",
+            "source": "sspdomain.com”,
+            "mm": "3”,
+                "uids": [
+                    {
+                        "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+			"atype": "1"
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+Note that there is no <code>buyeruid</code> in the example above. If the SSP does not have an explicit agreement from the DSP that they want to receive matched IDs in the <code>buyeruid</code> field, they would simply forward the above request to the DSP (either removing the eids array if the DSP doesn’t want to receive IDs from graphvendor.com, or retaining the array if the DSP does want to receive IDs from graphvendor.com in eids). 
+
+##### Example 4: Request from SSP → DSP corresponding to the above request from pub, where the SSP has set the DSP’s buyeruid based on a hosted cookie match table and the bridged ID they received from the pub (based on prior arrangement with the DSP only)
+
+If an arrangement exists to put a matched DSP ID in <code>buyeruid</code>, that would be done as follows:
+
+<b>NOTE:</b> This method should be used ONLY in cases where a previous arrangement has been made directly between the sell-side and the DSP to express a matched ID in <code>buyeruid</code>. If no agreement is in place, the ids in question should be sent in the EIDS array. 
+
+```
+{
+"user": {
+    "buyeruid": "3d89748d-74bb-44f1-9908-298090dc2944",
+    "eids": [
+        {
+            "matcher": "graphvendor.com",
+            "inserter": "pub.com",
+            "source": "sspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
+		    "atype": "1"
```

Comment:
again, this should be "atype":3, I believe.

Reply:
done